### DISCUSSION TITLE
Database owner nextcloud not allowed when updating from 8.3.0 to 9.0.1

### DISCUSSION THREAD

**Discussion by yannik-b** [nextcloud/all-in-one#4956] [Questions] [answered]
### Steps to reproduce
1. Have a running Nextcloud instance with helm chart in version 8.3.0.
2. Try an update to helm chart 9.0.1 as stated in the documentation.

### Expected behavior

After a few minutes the Nextcloud should be accessable via the web interface with the new version.

### Actual behavior

Nextcloud instance does not recover. In the logs of the database container, I got the following:

```
sed 's|Owner:||;s| ||g'
grep -a 'Name: oc_appconfig; Type: TABLE; Schema: public; Owner:' /mnt/data/database-dump.sql++ grep -oP 'Owner:.*$'

DB_OWNER=nextcloud
'[' nextcloud = nextcloud ']'
The database import failed. Please restore a backup and try again.
For further clues on what went wrong, look at the logs above.
```

In a earlier try, I got the issue database import failed with the notice that the database owner nextcloud must not be nextcloud or something like this.

### Host OS

Debian 12, Kubernetes used

#### Nextcloud AIO version 

AIO 9.0.1

#### Current channel 

stable

**Comment by szaimen** [comment, marked as answer]
> I got the issue database import failed with the notice that the database owner nextcloud must not be nextcloud or something like this.

Yes, this is correct. Not sure how you changed the owner? 

Can you run `grep -a 'Name: oc_appconfig; Type: TABLE; Schema: public; Owner:'` on the database dump and post the output here?

**Reply by yannik-b** [reply]
The output is 
```
sudo grep -a 'Name: oc_appconfig; Type: TABLE; Schema: public; Owner:' /path/to/database-dump.sql
-- Name: oc_appconfig; Type: TABLE; Schema: public; Owner: nextcloud
```

And no, I don't know how I might have changed the owner of the dump.

**Reply by szaimen** [reply]
I guess you need to adjust the database dump now and adjust `Owner: nextcloud` everywhere to `Owner: oc_nextcloud` so that the import works...

**Reply by yannik-b** [reply]
Yeah, that worked. Now I can enjoy an updated version :) 

But do you know how this happened? May it happen again for the next update?
